src/test/java/org/cactoos/text/ComparableTextTest.java

```diff
@@ -49,23 +53,107 @@ public void comparesWithASubtext() throws Exception {
                     new SubText("from here to there", 5)
                 )
             ),
-            Matchers.is(0)
-        );
+            new IsEqual<>(
```

Comment:
@ilyakharlamov Please, put this in the same line, something like `new IsEqual<>(0)`.